packages/dd-trace/src/config.js

```diff
@@ -806,31 +544,285 @@ ken|consumer_?(?:id|key|secret)|sign(?:ed|ature)?|auth(?:entication|orization)?)
     tagger.add(tags, DD_TRACE_TAGS)
     tagger.add(tags, DD_TRACE_GLOBAL_TAGS)
 
-    this._setString(env, 'service', DD_SERVICE || DD_SERVICE_NAME || tags.service)
+    this._setValue(env, 'appsec.blockedTemplateHtml', maybeFile(DD_APPSEC_HTTP_BLOCKED_TEMPLATE_HTML))
+    this._setValue(env, 'appsec.blockedTemplateJson', maybeFile(DD_APPSEC_HTTP_BLOCKED_TEMPLATE_JSON))
+    this._setBoolean(env, 'appsec.customRulesProvided', !!DD_APPSEC_RULES)
```

Comment:
This is still not fixed
```suggestion
    if (DD_APPSEC_RULES) this._setBoolean(env, 'appsec.customRulesProvided', true)
```